src/ascend.js

```diff
@@ -2,14 +2,13 @@ var _curry1 = require('./internal/_curry1');
 
 
 /**
- * Makes an "ascending" comparator function out of a function that returns a value
- * that can be compared
+ * Makes an ascending comparator function out of a function that returns a value
+ * that can be compared with `<` and `>`.
  *
  * @func
  * @memberOf R
- * @since v0.23.0
  * @category Function
- * @sig (a -> Number | String) -> (a, b -> Number)
+ * @sig Ord b => (a -> b) -> (a, a -> Number)
```

Comment:
to reduce ambiguity in the sig, please group the arguments to the returned function, i.e.

`... ((a, a) -> Number)`